projects/igniteui-angular/src/lib/navbar/navbar.component.html

```diff
@@ -1,8 +1,19 @@
 <nav class="igx-navbar" role="navigation" [attr.aria-labelledby]="titleId">
```

Comment:
I think we can get rid of this wrapping container. All of the attributes can be directly bound to the host element.

Reply:
Are you okay to track that one with another issue, since it's a bit different from the original?

Reply:
Well, if you insist. It will just create more overhead for an otherwise very simple change.